preprocessing/create_fashion_gen_retrieval_test_data.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+import pandas as pd
+import tensorflow as tf
+
+import utils
+
+
+"""
+The first 2 rows of txt_info file.
+
+86605,86605_0,JACKETS & COATS,7,DENIM JACKETS,39,Denim-like jogg jacket in blue. Fading and whiskering throughout. Spread collar. Copper tone button closures at front. Flap pockets at chest with metallic logo plaque. Seam pockets at sides. Cinch tabs at back waistband. Single button sleeve cuffs. Tone on tone stitching.
+86605,86605_1,JACKETS & COATS,7,DENIM JACKETS,39,Denim-like jogg jacket in blue. Fading and whiskering throughout. Spread collar. Copper tone button closures at front. Flap pockets at chest with metallic logo plaque. Seam pockets at sides. Cinch tabs at back waistband. Single button sleeve cuffs. Tone on tone stitching.
+"""
+
+txt_info_filename = f'gs://mmt/raw_data/fashion_gen/full_valid_info.txt'
```

Comment:
Add a comment to replace with actual folders if someone clones this repo and they want to run this.